canonical-websites/netplan.io
# added documentation and design

## Done

- added documentation and design
- added some design elements to the overview

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [0.0.0.0:8024](http://0.0.0.0:8024/)
- see that it looks a bit like the [copy doc](https://docs.google.com/document/d/1ZmYO1DIO9bUlaYvThUWF5vSSp5OVsmBoa8nRu15usm0/edit#)
